packages/devextreme/scss/widgets/base/_menuBase.scss

```diff
@@ -89,5 +94,13 @@
 
   &.dx-overlay-content .dx-scrollable {
     overflow: visible;
+
+    .dx-scrollable-content {
+      transform: none !important; // stylelint-disable-line declaration-no-important
```

Comment:
Let's add a note describing what is this for to minimize possible removing in the future during some refactoring